backend/src/app.js

```diff
@@ -28,6 +28,7 @@ app.use(bodyParser.raw());
 
 // routes
 app.use('/api/listings', require('./routes/listings'));
+app.use('/api/user', require('./routes/preferences'));
```

Comment:
I think the endpoint should still be `/users/` ?

Reply:
And all the requests would be done on the `/api/users/:userId/preferences` path or `/api/users/:userId/recommendations`

Reply:
Sure! Will change to this